tidb-cloud/premium/tidb-cloud-auditing-premium.md

```diff
@@ -0,0 +1,282 @@
+        }
+        ```
+
+        In the template, `<Your S3 bucket ARN>` is the Amazon Resource Name (ARN) of your S3 bucket where the audit log files are to be written. You can go to the **Properties** tab in your S3 bucket and get the ARN value in the **Bucket Overview** area. In the `"Resource"` field, you need to add `/*` after the ARN. For example, if the ARN is `arn:aws:s3:::tidb-cloud-test`, you need to configure the value of the `"Resource"` field as `"arn:aws:s3:::tidb-cloud-test/*"`.
+
+3. Go to **IAM** > **Access Management** > **Roles**, and then check whether a role whose trust entity corresponds to the TiDB Cloud Account ID and the External ID that you recorded earlier already exists.
+
+    - If yes, record the matched role for later use.
+    - If not, click **Create role**, select **Another AWS account** as the trust entity type, and then enter the TiDB Cloud Account ID value into the **Account ID** field. Then, choose the **Require External ID** option and enter the TiDB Cloud External ID value into the **External ID** field.
+
+4. In **IAM** > **Access Management** > **Roles**, click the role name from the previous step to go to the **Summary** page, and then take the following steps:
+
+    1. Under the **Permissions** tab, check whether the recorded policy with the `s3:PutObject` write-only permission is attached to the role. If not, choose **Attach Policies**, search for the needed policy, and then click **Attach Policy**.
+    2. Return to the **Summary** page and copy the **Role ARN** value to your clipboard.
+
+#### Step 3. Enable audit logging
+
+In the TiDB Cloud console, go back to the **Database Audit Log Storage Configuration** dialog box where you got the TiDB Cloud account ID and the External ID values, and then take the following steps:
+
+1. In the **Bucket URI** field, enter the URI of your S3 bucket where the audit log files are to be written.
+2. In the **Bucket Region** drop-down list, select the AWS region where the bucket is located.
+3. In the **Role ARN** field, fill in the Role ARN value that you copied in [Step 2. Configure Amazon S3 access](#step-2-configure-amazon-s3-access).
+4. Click **Test Connection and Next** to verify whether TiDB Cloud can access and write to the bucket.
+
+    If it is successful, **The connection is successful** is displayed. Otherwise, check your access configuration.
+
+5. Click **Enable** to enable audit logging for the instance.
+
+    TiDB Cloud is ready to write audit logs for the specified instance to your Amazon S3 bucket.
+
+> **Note:**
+>
+> - After enabling audit logging, if you make any new changes to the bucket URI, location, or ARN, you must click **Test Connection** again to verify that TiDB Cloud can connect to the bucket. Then, click **Enable** to apply the changes.
+> - To remove TiDB Cloud's access to your Amazon S3, simply delete the trust policy granted to this instance in the AWS Management Console.
+
+### Enable audit logging for TiDB on Alibaba Cloud
```

Comment:
Not sure if we need to wrap this section with CustomContent, just in case we forget to do so when Premium goes into public preview in the future.

```
<CustomContent language="en,zh">

</CustomContent>
```